### PR TITLE
Dinesh101041 issue template

### PR DESCRIPTION
**Pull request by Dinesh101041**
**Added a Issue template** 

Issue : #8

### CODE REVIEW

**Comment by zhanghang1989** [comment]
Got a linter error. Could you change the Tab with space for indent?

**Comment by Dinesh101041** [comment]
want to change the space in files right sir

File: .github/ISSUE_TEMPLATE/---bugs.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+<put code or diff here>
+```
+2. What exact command you run:
+3. __Full logs__ or other relevant observations:
+```
+<put logs here>
+```
+4. please simplify the steps as much as possible so they do not require additional resources to
+	 run, such as a private dataset.
```
[the remark this answers is not in the document]
Comment (Dinesh101041) [reply .github/ISSUE_TEMPLATE/---bugs.md]: changed sir

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Documentation
+about: Report a documentation issue
+title: ''
+labels: documentation
+assignees: ''
+
+---
+
```
Comment (zhanghang1989) [inline .github/ISSUE_TEMPLATE/documentation.md:9]: Could you remove two tailing empty lines?

File: .github/ISSUE_TEMPLATE/--u0001f680feature-request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (zhanghang1989) [inline .github/ISSUE_TEMPLATE/--u0001f680feature-request.md:1]: Could you rename the file to `feature-request.md`?

File: .github/ISSUE_TEMPLATE/---bugs.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (zhanghang1989) [inline .github/ISSUE_TEMPLATE/---bugs.md:1]: There're issues with the filenames. Could you rename the file to `bugs.md`

**Review by zhanghang1989 — COMMENTED** [review]
There're issues triggered when trying to merge the code. Could you remove `--` in the file names? Thanks!

File: .github/ISSUE_TEMPLATE/---bugs.md (hunk 4 above)
Comment (Dinesh101041) [reply .github/ISSUE_TEMPLATE/---bugs.md]: all done sir

**Comment by zhanghang1989** [comment]
Thanks for your contributions!

**Comment by Dinesh101041** [comment]
Thank you for merging sir

**Pull request closed without merging**